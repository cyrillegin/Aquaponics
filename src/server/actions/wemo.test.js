import toggleWemo from './wemo';

jest.mock('child_process', () => ({
  exec: command => {
    expect(command).toMatchSnapshot();
  },
}));
describe('wemo', () => {
  it('should test the function', () => {
    toggleWemo({ metaData: '{"name":"test", "event":"test"}' });
  });
});
