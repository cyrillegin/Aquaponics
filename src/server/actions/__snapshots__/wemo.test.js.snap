// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wemo should test the function 1`] = `"wemo switch test test"`;
